### DISCUSSION TITLE
Test Review Request

### DISCUSSION THREAD

**Discussion by abadger** [ansible-collections/ansible-inclusion#2] [Resolved reviews]
I would like to have the  example.fake collection reviewed for potential inclusion in the ansible package.

* Galaxy URL: https://galaxy.ansible.com/example/fake
* Git repo: https://github.com/example/fake

**Comment by abadger** [comment]
* Reviewers will look at the checklist and the collection itself to determine if it should be approved for inclusion.
* Once they're satisfied, they should explicitly say that they approve it here.
* Then open a PR in https://github.com/ansible-community/ansible-build-data to add the collection to the next version of ansible.
  * Include a link to this review in the PR.
* Someone from releng (people who have commit to ansible-build-data) will then merge the PR and recategorize this review request from "New collection reviews" to "Resolved reviews".
  * Recategorizing allows us to refer to the review later if there's a problem with the collection.

**Reply by abadger** [reply]
New:
* The ansible-build-data PR should also add the collection owners to the VERSION/collection-meta.yml  file.

**Comment by abadger** [comment]
A good review should include:
* Link to the exact collection version downloaded from galaxy
* Checklist with all boxes checked or a comment as to why it was not applicable

**Comment by abadger** [comment]
Things TODO:
* Add templates for submission to this page.
* Can we add a template for reviews that the reviewer can select from or do we need to make that a page that they can cut and paste from?
* template in the ansible-build-data repo for submitting a PR to add a collection

**Reply by Andersson007** [reply]
+1
> Include a link to this review in the PR.
>
> New:
> 
> * The ansible-build-data PR should also add the collection owners to the VERSION/collection-meta.yml  file.

and include the both points there (and other necessary / highly recommended things) as reminders
